src/components/BottomNav/BottomNav.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useRef, useState } from 'react'
+import { useMatch } from 'react-router-dom'
+import { CSSTransition } from 'react-transition-group'
+
+import { navItems } from '@/config/nav'
+import { transitions } from '@/shared/theme'
+
+import { Container, NavLink, NavTitle } from './BottomNav.style'
+
+export const BottomNav = () => {
+  const [open, setOpen] = useState(false)
+  const match = useMatch
+  const pageYOffsetRef = useRef<number | null>(null)
+
+  const toggleNavbar = () => {
+    const scrollRange = pageYOffsetRef.current !== null ? window.pageYOffset - pageYOffsetRef.current : 0
+    if (scrollRange >= 128) {
+      setOpen(true)
+    }
+    if (scrollRange <= -128) {
+      setOpen(false)
+    }
+    if (Math.abs(scrollRange) >= 128 || pageYOffsetRef.current === null) {
+      pageYOffsetRef.current = window.pageYOffset
+    }
+  }
+
+  useEffect(() => {
+    window.addEventListener('scroll', toggleNavbar)
+
+    return () => {
+      window.removeEventListener('scroll', toggleNavbar)
+    }
+  }, [])
+
+  return (
+    <CSSTransition
+      in={open}
+      timeout={parseInt(transitions.timings.routing)}
+      classNames="bottom-nav"
+      unmountOnExit
+      mountOnEnter
+    >
+      <Container>
+        {navItems.map((item) => (
+          <NavLink to={item.to} key={`bottomLink-${item.to}`} active={match(item.to)}>
+            {item.icon}
+            <NavTitle>{item.name}</NavTitle>
```

Comment:
Font is different in the design: 
![image](https://user-images.githubusercontent.com/51168865/135405335-e412d6cb-6bc0-48fc-86c2-2229103cf6b1.png) You should use our `Text` with `button3 ` variant here.

But... on the other hand, we could stick to `Inter` here, because it looks weird when I applied `PxGrotesk`. Let's ask designer about this, maybe?

Reply:
Ok, talked to Adam.  You should change this to `PxGrotesk`, also you should update the font in desktop navigation( you'll need to update `::after` pseudo-element probably)

Reply:
@drillprop knowing that soon we'll be replacing `PxGrotesk` with a completely different typeface altogether, I'd suggest sticking with `PxGrotesk` for now, to keep the implementation and design consistent when this happens.